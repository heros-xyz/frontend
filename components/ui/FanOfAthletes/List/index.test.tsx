import { render } from "@testing-library/react";
import YourAthletesList from ".";

it("renders homepage unchanged", () => {
  const { container } = render(
    <YourAthletesList
      atheleList={[
        {
          avatar:
            "https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png",
          name: "Matt Prior",
          description: " Bronze Tier Subscribed",
          joinedDate: "22/12/2022",
        },
        {
          avatar:
            "https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png",
          name: "Matt Prior",
          description: " Bronze Tier Subscribed",
          joinedDate: "22/12/2022",
        },
        {
          avatar:
            "https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png",
          name: "Matt Prior",
          description: " Bronze Tier Subscribed",
          joinedDate: "22/12/2022",
        },
        {
          avatar:
            "https://cdn.iconscout.com/icon/free/png-256/avatar-370-4ddddd56322.png",
          name: "Matt Prior",
          description: " Bronze Tier Subscribed",
          joinedDate: "22/12/2022",
        },
      ]}
    />
  );
  expect(container).toMatchSnapshot();
});
