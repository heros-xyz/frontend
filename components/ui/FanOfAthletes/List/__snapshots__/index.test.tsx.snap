// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    athelelist="[object Object],[object Object],[object Object],[object Object]"
    class="css-0"
  >
    <h2
      class="chakra-heading css-e9d1fw"
    >
      No results.
    </h2>
  </div>
</div>
`;
